assets/app/views/site/edit.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs');
 var Backbone = require('backbone');
 var _ = require('underscore');
 
-var templateHtml = fs.readFileSync(__dirname + '/../templates/SiteEditTemplate.html').toString();
+var templateHtml = fs.readFileSync(__dirname + '/../../templates/site/edit.html').toString();
```

Comment:
It is my understanding that `require()` doesn't work for files that aren't `.js` or `.json`. When I attempt to use `require` I get errors.

Right now we have `browserify` use the `brfs` to inline the contents of those files into the JS files during the build step